daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -650,61 +652,65 @@ object Speedy {
   }
 
   /** The function has been evaluated to a value, now start evaluating the arguments. */
+  def executeApplication(machine: Machine, vfun: SValue, newArgs: Array[SExpr]): Unit = {
```

Comment:
```suggestion
  private def executeApplication(machine: Machine, vfun: SValue, newArgs: Array[SExpr]): Unit = {
```

Reply:
This can't be `private` because it is also used in `SExpr.scala`